City-of-Helsinki/helsinki-paatokset
# Updated core + all modules + changed helfi_* module settings

Core 9.1.7 -> 9.1.8

Helfi_* modules haven been updated. Install location has been moved from custom to contrib directory. 

To test:
- `make build; make post-install; drush-cr;`
- Most helfi -prefixed modules should have moved from `public/modules/custom` to `public/modules/contrib`
- Verify that site still works
